src/AdminForm.php

```diff
@@ -427,18 +427,18 @@ private function buildContactTab($n, $c) {
       }
     }
   }
-
+fa-id-badge
   /**
```

Comment:
This line/note is what's currently failing your test 🙂

Reply:
I'll have that changed!